Add note on enclave pricing to guide (#721)
vespa-engine/cloud: en/enclave.md

Enclaves currently only support AWS, but GCP support will be available soon.

> Please note: Since your Vespa Cloud Enclave resources run in _your_ account, this will incur resource costs from your cloud provider in _addition_ to the Vespa Cloud costs.

Reviewer: ```suggestion
Note: As the Vespa Cloud Enclave resources run in _your_ account, this incurs resource costs from your cloud provider in _addition_ to the Vespa Cloud costs.
```
